### PR TITLE
Group fields

### PR DESCRIPTION
**Pull request by tomasperezv**
### What is the goal?

Group sequentially items inside of a wrapper element.

### How is being implemented?

Added a new method `group` that it's called automatically by `getFragment` and will be activated if we pass the parameter `groups` to the options.

### Reviewers

@jobandtalent/frontend

### CODE REVIEW

File: src/javascript/dom/builder/factory.js [hunk 1]
```diff
@@ -18,6 +18,9 @@ class FragmentFactory {
   /**
    * @method create
    * @param {Object} params
+   *  @type {String} type
+   *  @type {Object} data
+   *  @type {Object} groups
```
Comment (tomasperezv) [inline src/javascript/dom/builder/factory.js]: Just extending a bit missing documentation.

File: src/javascript/dom/builder/factory.js [hunk 2]
```diff
@@ -34,6 +37,9 @@ class FragmentFactory {
   /**
    * @method _instantiate
    * @param {Object} params
+   *  @type {String} type
```
Comment (tomasperezv) [inline src/javascript/dom/builder/factory.js]: Same as previous comment.

File: src/javascript/dom/builder/fragment/base.js [hunk 3]
```diff
@@ -59,6 +60,8 @@ class Base {
 
     this._renderLocators();
     this._decorate();
+    this.group();
```
Comment (tomasperezv) [inline src/javascript/dom/builder/fragment/base.js]: I made `group` explicitly public to be able to use it externally

**Comment by montiu** [comment]
👍

**Comment by calderon** [comment]
👌

**Pull request merged**